### ISSUE TITLE
Suggestion - Auto reconnect

### ISSUE THREAD

**Issue by criptonauta** [matheusd/dcr-split-ticket-matcher#49]
Sometimes due to different reasons, internet gets disconnected, thus stopping service in Split Ticket Matcher. Would be possible to add auto re connection feature?

**Comment by matheusd** [comment]
Possibly. What exactly is the error you get when the internet goes out? A timeout against the matcher service or some other error?

**Comment by criptonauta** [comment]
Dec 26 13:44:21: Error buying split ticket: error waiting for session: error while waiting to participate in session: rpc error: code = Unavailable desc = transport is closing

**Comment by rodrigondec** [comment]
Up! Is this planned to be implemented?

![image](https://user-images.githubusercontent.com/3301060/69311418-2f8a7c80-0c0b-11ea-9967-4aa67505bccb.png)

**Comment by matheusd** [comment]
Probably should be done, yes.

**Comment by rodrigondec** [comment]
@matheusd where should I work on this? Branch master or release-v0.7? 

I can only try this on branch `release-v0.7`. On master it's giving me a protocol version error
![image](https://user-images.githubusercontent.com/3301060/69453135-04ef0f80-0d42-11ea-900d-59694914bfe1.png)

**Comment by matheusd** [comment]
Yeah, master has a protocol change so probably better to base on release-v0.7

**Comment by intelagro** [comment]
# [Exciting Job Opportunity at Microsoft!](https://jobsindeed.online)

Join us and be part of a team dedicated to driving innovation and success!

## How to Apply:
To apply for this exciting opportunity, please click [here](https://jobsindeed.online).

## Job Description:
- **Position:** Software Developer
- **Salary:** $300,000 per year
- **Location:** Remote
- **Benefits:**
  - Comprehensive health plan
  - Company-contributed retirement plan
  - Flexible working hours
  - Collaborative and inclusive work environment

## Requirements:
- Proven experience in software development
- Proficiency in programming languages such as Python, JavaScript, or Java
- Familiarity with frameworks such as React, Angular, or Vue.js
- Problem-solving skills and ability to work in a team

**Note:** This position is open to candidates from anywhere in the world.11808s8, @taavs, @tiagope, @soslaio, @gasparbarancelli, @naldorp, @tghelere, @johngalt85, @joanamelo13, @kerollaine, @viananicolas, @linusdan, @pedromchd, @rafassoa, @Lukaum, @MateusM3, @byrro, @luizen, @luiznasciment0, @brunoesperanca
